src/signatures.rs

```diff
@@ -127,10 +132,18 @@ impl Serializable<64> for Signature {
 pub(crate) fn challenge_hash(
     R: &JubJubExtended,
     message: BlsScalar,
+    pk: PublicKey,
```

Comment:
I think the parameter should be in the same order as for the hash input:
`R, pk, message`